### PR TITLE
97 switch from portray to plain mkdocs + mkdocstrings

### PR DESCRIPTION
**Pull request by tjadevries**
Did not yet test the github actions but I am pretty confident that they work.

We now have a more standard way to accomplish the same as what we had but now with correct code reference.

### CODE REVIEW

File: .github/workflows/build_deploy_docs_develop.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
 
       - name: Build html
         run:
-          poetry run portray as_html --overwrite -o site
+          poetry run mkdocs build -f ./docs/mkdocs.yml -d ../site"
```
Comment (JJdeVries) [inline .github/workflows/build_deploy_docs_develop.yml:52]: Is the trailing `"` correct? seems rather odd

File: .github/workflows/build_deploy_docs_release.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ jobs:
 
       - name: Build html
         run:
-          poetry run portray as_html --overwrite -o site
+          poetry run mkdocs build -f ./docs/mkdocs.yml -d ../site"
```
Comment (JJdeVries) [inline .github/workflows/build_deploy_docs_release.yml:55]: same here

**Review by JJdeVries — CHANGES_REQUESTED** [review]

File: .github/workflows/build_deploy_docs_develop.yml (hunk 1 above)
Comment (tjadevries) [reply .github/workflows/build_deploy_docs_develop.yml:52]: You're right ofcourse

File: .github/workflows/build_deploy_docs_release.yml (hunk 2 above)
Comment (tjadevries) [reply .github/workflows/build_deploy_docs_release.yml:55]: it's gone...

**Review by JJdeVries — APPROVED** [review]

**Pull request merged**